SPECS/kata-containers/kata-containers.spec

```diff
@@ -67,6 +67,7 @@ popd
 %{kata_bin}/kata-runtime
 
 %{defaults_kata}/configuration.toml
+%{defaults_kata}/configuration-clh-debug.toml
```

Comment:
Does naming this to configuration-debug.toml make our life easier anywhere else?

Reply:
I don't see any code change required due to this name. It comes out of the kata-containers common.sh file names.